src/test/scala/com/fulcrumgenomics/sopt/cmdline/ClpArgumentDefinitionPrintingTest.scala

```diff
@@ -81,4 +87,8 @@ class ClpArgumentDefinitionPrintingTest extends UnitSpec with BeforeAndAfterAll
     printArgumentUsage(longName, shortName, theType,   Some("+"),      description).startsWith(s"-${shortName.get} $theType+, --$longName=$theType+") shouldBe true
     printArgumentUsage(longName, shortName, theType,   Some("{0,20}"), description).startsWith(s"-${shortName.get} $theType{0,20}, --$longName=$theType{0,20}") shouldBe true
   }
+
+  "ClpArgumentDefinitionPrinting.possibleValues" should "find the possible values in a sealed trait/case object hierarchy" in {
+    ClpArgumentDefinitionPrinting.possibleValues(classOf[Foo]) shouldBe "Options: Hi, Lo, Whee."
```

Comment:
Only improvement I can possibly think of would be to have it output `Options: Hi, Lo, and Whee.`